src/MapTiles.jl

```diff
@@ -145,20 +145,20 @@ module MapTiles
         ])
     end
 
-    project(basemap::BaseMap, geom::GeoInterface.AbstractPoint) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractPointTrait) = 
         project(GeoInterface.Point, basemap, geom)
-    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiPoint) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiPointTrait) = 
         project(GeoInterface.MultiPoint, basemap, geom)
-    project(basemap::BaseMap, geom::GeoInterface.AbstractLineString) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractLineStringTrait) = 
         project(GeoInterface.LineString, basemap, geom)
-    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiLineString) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiLineStringTrait) = 
         project(GeoInterface.MultiLineString, basemap, geom)
-    project(basemap::BaseMap, geom::GeoInterface.AbstractPolygon) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractPolygonTrait) = 
         project(GeoInterface.Polygon, basemap, geom)
-    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiPolygon) = 
+    project(basemap::BaseMap, geom::GeoInterface.AbstractMultiPolygonTrait) = 
         project(GeoInterface.MultiPolygon, basemap, geom)
 
-    function project(basemap::BaseMap, geom::GeoInterface.AbstractGeometry)
+    function project(basemap::BaseMap, geom::GeoInterface.AbstractGeometryTrait)
         gtype = GeoInterface.geotype(geom)
         if gtype == :Point
             return project(GeoInterface.Point, basemap, geom)
```

Comment:
There is still some more old GeoInterface code here. `geotype` and `GeoInterface.Point` both no longer exist.

Reply:
@visr is there a 1-2-1 replacement for `geotype` ?

Reply:
Not quite the same, but there is `geomtrait`, which doesn't return a Symbol but rather a trait that we can dispatch on.